assets/js/index.js

```diff
@@ -0,0 +1,11 @@
+// Save the selected day to localStorage
```

Comment:
Can we also display month and year, for example: December 2023 (let's remember we want to be able view current month of the year so it would be good not to hardcode it)